examples/triangle-example.ts

```diff
@@ -0,0 +1,253 @@
+
+        this._defaultFBO = new DefaultFramebuffer(context, 'DefaultFBO');
+        this._defaultFBO.initialize();
+        this._defaultFBO.bind();
+
+        const gl = context.gl;
+
+        this._buffer = gl.createBuffer();
+        gl.bindBuffer(gl.ARRAY_BUFFER, this._buffer);
+        gl.bufferData(gl.ARRAY_BUFFER, this._triangle, gl.STATIC_DRAW);
+
+
+        const vert = new Shader(context, gl.VERTEX_SHADER, 'mesh.vert');
+        vert.initialize(require('./data/mesh.vert'));
+        const frag = new Shader(context, gl.FRAGMENT_SHADER, 'mesh.frag');
+        frag.initialize(require('./data/mesh.frag'));
+
+
+        this._program = new Program(context, 'TriangleProgram');
+        this._program.initialize([vert, frag], false);
+
+        this._program.link();
+        this._program.bind();
+
+        this._vertexLocation = this._program.attribute('a_vertex', this._vertexLocation);
+        this._uvCoordLocation = this._program.attribute('a_texCoord', this._uvCoordLocation);
+
+
+        gl.vertexAttribPointer(
+            this._vertexLocation,   // Attribute Location
+            3,                      // Number of elements per attribute
+            gl.FLOAT,               // Type of elements
+            gl.FALSE,
+            5 * Float32Array.BYTES_PER_ELEMENT, // Size of an individual vertex
+            0, // Offset from the beginning of a single vertex to this attribute
+        );
+
+        gl.vertexAttribPointer(
+            this._uvCoordLocation,   // Attribute Location
+            2,                      // Number of elements per attribute
+            gl.FLOAT,               // Type of elements
+            gl.FALSE,
+            5 * Float32Array.BYTES_PER_ELEMENT, // Size of an individual vertex
+            3 * Float32Array.BYTES_PER_ELEMENT, // Offset from the beginning of a single vertex to this attribute
+        );
+
+        gl.enableVertexAttribArray(this._vertexLocation);
+        gl.enableVertexAttribArray(this._uvCoordLocation);
+
+
+        this._uViewProjection = this._program.uniform('u_viewProjection');
+        const identity = mat4.identity(mat4.create());
+        gl.uniformMatrix4fv(this._program.uniform('u_model'), gl.FALSE, identity);
+        gl.uniform1i(this._program.uniform('u_texture'), 0);
+        gl.uniform1i(this._program.uniform('u_textured'), false);
+
+
+        this._texture = new Texture2D(context, 'Texture');
+        this._texture.initialize(1, 1, gl.RGB, gl.RGB, gl.UNSIGNED_BYTE);
+        this._texture.wrap(gl.REPEAT, gl.REPEAT);
+        this._texture.filter(gl.LINEAR, gl.LINEAR_MIPMAP_LINEAR);
+        this._texture.maxAnisotropy(Texture2D.MAX_ANISOTROPY);
+
+        this._texture.fetch('./data/triangle-texture.webp', false).then(() => {
+            const gl = context.gl;
+
+            this._program.bind();
+            gl.uniform1i(this._program.uniform('u_textured'), true);
+            console.log('enabled texture.');
```

Comment:
Please remove debug output or at least hide it behind certain verbosity levels (see https://github.com/cginternals/webgl-operate/blob/master/source/auxiliaries.ts#L89).